repos/system_upgrade/el7toel8/actors/checksendmail/tests/component_test.py

```diff
@@ -0,0 +1,53 @@
+from leapp.snactor.fixture import current_actor_context
+from leapp.models import InstalledRedHatSignedRPM, RPM, SendmailMigrationDecision, DaemonList, TcpWrappersFacts
+from leapp.reporting import Report
+from leapp.libraries.actor.library import SendmailConfDir
+import os
```

Comment:
Please put `import os` before leapp imports with a single newline as a logical separator
Also please sort imports alphabetically within logical block (I think we should definitely add  this rule to pep8 check)

Reply:
It should be fixed by @examon commits.